3rdpartylibrary/google-maps-android-cmgn/README.md

```diff
@@ -0,0 +1,4 @@
+== Google Maps Android CMGN ==
+
+Google Maps Code which has been modified to use CMAPI components for things like geoposition, etc.
+This module is built as a jar.
```

Comment:
Is this being _built_ as a jar or _included_ as a jar?

Reply:
The result of the build is a jar.